app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
For more readability consider using type annotation. 
```suggestion
from typing import List
def go_to_cafe(friends: List[dict], cafe:Cafe) -> str:
```